migrations/sqls/20210219000521-create-questions-and-answers-tables-up.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE customServiceQuestions (
```

Comment:
This file shouldn't be showing up again, since it's already in the master branch. This likely indicates that you need to merge master into your branch.

```sh
# Make sure you have a clean git status. If you don't, stash or commit any changes.
git status

git checkout master
git pull
git checkout issue-753
# If this results in merge conflicts, resolve the conflicts
git merge master
git push
```